app/mailers/checkout_notifier_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class CheckoutNotifierMailer < ApplicationMailer
+  default :from => 'railscasts@gmail.com'
+
+  def send_checkout_email(user)
+    mail( :to => user.email,
+    :subject => 'Thanks for checkout items for our amazing website' )
```

Comment:
`mail(to: user.email, subject: 'Thanks for checkout items for our amazing website' )`